### PR TITLE
Crawler for Tableau [sc-1336]

### PR DESCRIPTION
**Pull request by alyiwang**
### 🤔 Why?

To add Tableau to our support dashboard platforms

### 🤓 What?

- Add crawler for tableau dashboard using REST API
- Fetches dashboards and charts. No data source connection yet
- Support both user-password and access token (recommended) auth

### 🧪 Tested?

local run successfully against https://10ax.online.tableau.com/#/site/metaphordev451768 , using either password or token. 
Collected two dashboards and 15 charts.

### CODE REVIEW

File: metaphor/tableau/extractor.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        dashboard_info = DashboardInfo(
+            title=f"{workbook.project_name}.{workbook.name}",
+            description=workbook.description,
+            url=workbook.webpage_url,
+            charts=charts,
+            view_count=float(total_views),
+        )
+
+        dashboard = Dashboard(
+            logical_id=DashboardLogicalID(dashboard_id=workbook.id),
```
Comment (mars-lan) [inline metaphor/tableau/extractor.py:108]: Is `workbook.id` enough to guarantee global uniqueness?

**Review by mars-lan — COMMENTED** [review]
Great work!

File: metaphor/tableau/extractor.py (hunk 1 above)
Comment (alyiwang) [reply metaphor/tableau/extractor.py:108]: It's a UUID, so should be globally unique

File: metaphor/tableau/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Config File
+
+Create a YAML config file based on the following template.
+
+### Required Configurations
+
+```yaml
+server_url: <tableau_server_url>  // e.g. https://10ay.online.tableau.com
+site_name: <site_name>
```
[the remark this answers is not in the document]
Comment (alyiwang) [reply metaphor/tableau/README.md]: It's like an account, which contains users and contents. Added a comment in the README.

**Review by mars-lan — APPROVED** [review]

**Comment by mars-lan** [comment]
Please also update the top-level README in your next PR thanks.

**Pull request merged**